tests/Aspire.Hosting.Tests/Azure/AzureResourceExtensionsTests.cs

```diff
@@ -172,4 +173,36 @@ public void AddAzureCosmosDBWithEmulatorGetsExpectedImageTag(string imageTag)
         var actualTag = containerImageAnnotation.Tag;
         Assert.Equal(imageTag ?? "latest", actualTag);
     }
+
+    [Theory]
+    [InlineData(null)]
+    [InlineData(8081)]
+    [InlineData(9007)]
+    public void AzureEventHubsWithEmulatorGetsExpectedPort(int? port = null)
```

Comment:
Do we need to add integration tests that actually spin up the emulator?

Reply:
Yes, I want to do that (started with the easy ones) from existing emulators.